### PR TITLE
Improvements:

### PR DESCRIPTION
**Pull request by kaleofduty**
- Add chain ids for Polygon zkEVM
- Increase maximum OCR2 offchain config size
- Upgrade to go version 1.20
- Bump various dependencies, most notably go-ethereum
- Rename offchainreporting2 -> offchainreporting2plus to prepare for future versions of OCR
- OCR3 alpha (DO NOT USE IN PROD)

Based on cd5b9538136045ea77ec01d637fa86bdb043adc3

### CODE REVIEW

File: offchainreporting2plus/internal/managed/managed_mercury_oracle.go [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+				"ManagedMercuryOracle: error during reportingPlugin.Close()",
+			)
+
+			// if err := validateReportingPluginLimits(merucuryPluginInfo.Limits); err != nil {
+			// 	logger.Error("ManagedMercuryOracle: invalid ReportingPluginInfo", commontypes.LogFields{
+			// 		"error":               err,
+			// 		"reportingPluginInfo": reportingPluginInfo,
+			// 	})
+			// 	return
+			// }
```
Comment (samsondav) [inline offchainreporting2plus/internal/managed/managed_mercury_oracle.go:119]: commented code?

File: offchainreporting2plus/internal/ocr3/protocol/oracle.go [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+	chNetToPacemaker          chan<- MessageToPacemakerWithSender[RI]
+	chNetToReportGeneration   chan<- MessageToReportGenerationWithSender[RI]
+	chNetToReportFinalization chan<- MessageToReportFinalizationWithSender[RI]
+	childCancel               context.CancelFunc
+	childCtx                  context.Context
+	epoch                     uint64
+	subprocesses              subprocesses.Subprocesses
+}
+
+// TODO: This comment is outdated
```
Comment (samsondav) [inline offchainreporting2plus/internal/ocr3/protocol/oracle.go:77]: ??

File: offchainreporting2plus/internal/ocr3/protocol/persist/persist_transmission.go [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+			// 	}
+			// 	if err != nil {
+			// 		logger.ErrorIfNotCanceled(
+			// 			"PersistTransmission: error updating database",
+			// 			dbCtx,
+			// 			commontypes.LogFields{"error": err, "store": store},
+			// 		)
+			// 		return
+			// 	}
+			// }()
```
Comment (samsondav) [inline offchainreporting2plus/internal/ocr3/protocol/persist/persist_transmission.go:56]: ??

**Review by coventry — APPROVED** [review]
Only changes in whitespace and comments, as expected.

**Pull request merged**